### ISSUE TITLE
1.6.1 broken with multiple networks on Ubuntu

### ISSUE THREAD

**Issue by bartmichu** [zerotier/ZeroTierOne#1310]
My laptop is authorized on multiple networks but since last update I'm unable to join or leave them all.
Mind you I'm not trying to do this at once.

```shell
% sudo zerotier-cli leave A
200 leave OK

(wait a cauple of minutes)

% sudo zerotier-cli leave B
200 leave OK

(wait a cauple of minutes)

% sudo zerotier-cli leave C
0 leave timed out

% sudo zerotier-cli listnetworks          
Error connecting to the ZeroTier service: timed out

Please check that the service is running and that TCP port 9993 can be contacted via 127.0.0.1.
```

If I wait long enough (minutes) then I can get ```listnetworks``` to work, but ```leave``` or ```join``` never recovers - even after half an hour.

## Additional information 

**Desktop (please complete the following information):**
 - OS: Linux
 - OS/Distribution Version: Ubuntu 2010 on amd64
 - ZeroTier Version: 1.6.1
  
**Additional context**

On 1.4.6 I'm using a shell script to join/leave multiple networks almost simultaneously and it _never_ fails.
On 1.6.1 I'm unable to join/leave multiple networks by hand, it _always_ fails.

**Comment by guilhermecorreagv** [comment]
Typing `sudo systemctl restart zerotier-one` fixed it for me.

**Comment by bartmichu** [comment]
In my case restarting service or system is not fixing it.

**Comment by bartmichu** [comment]
Fixed in 1.6.2
